docs/apidiffs/1.10.0_vs_1.9.0/opentelemetry-api.txt

```diff
@@ -0,0 +1,150 @@
+Comparing source compatibility of  against 
```

Comment:
Does the `docs/apidiffs/current_vs_latest/` directory reset in a separate PR?

Reply:
Hmm might have forgotten git add again